rust_src/src/base64.rs

```diff
@@ -20,19 +21,9 @@ use crate::{
 };
 
 fn base64_encode_1(bytes: &[u8], line_break: bool, multibyte: bool) -> Result<String, ()> {
-    let config = if line_break {
-        // base64_crate::MIME, but with LF instead of CRLF
-        base64_crate::Config::new(
-            base64_crate::CharacterSet::Standard,
-            true, // pad
-            true, // strip whitespace
-            base64_crate::LineWrap::Wrap(76, base64_crate::LineEnding::LF),
-        )
-    } else {
-        base64_crate::STANDARD
-    };
+    let config = base64_crate::STANDARD;
```

Comment:
`pad` is still true for `STANDARD`?

Reply:
Yes for this version `base64 (v0.10.0)` (No padding version is provided as `STANDARD_NO_PAD`).